schema.sql

```diff
@@ -1,5 +1,16 @@
 /* Database schema to keep the structure of entire database. */
 
+-- CREATE TABLE animals (
+--     name varchar(100)
+-- );
+
```

Comment:
- [ ] Let's not use commented code. Leaving commented code in your codebase is a bad practice.

Reply:
Done